indico/util/date_time.py

```diff
@@ -315,57 +315,93 @@ def _format_pretty_datetime(dt, locale, tzinfo, formats):
     return _format_datetime(dt, fmt, tzinfo, locale)
 
 
-def format_pretty_date(dt, locale=None, tzinfo=None):
+def format_pretty_date(dt, locale=None, tzinfo=None, capitalize=True):
```

Comment:
```suggestion
def format_pretty_date(dt, locale=None, tzinfo=None, *, capitalize=True):
```

Boolean args should almost always be keyword-only